keyboards/geistmaschine/macropod/rules.mk

```diff
@@ -0,0 +1,7 @@
+ENCODER_MAP_ENABLE = yes
+
```

Comment:
Same comment from other PR about encoder map not being enabled at keyboard level.
```suggestion
```

Reply:
Sorry for applying the changes so late, have been busy in the past few days - I moved the encoder map enable to keymap level configs, and added an encoder map to board.c!